### PR TITLE
Adding documentation for config options provided by plugin

### LINKED ISSUE #230: Documentation for configuring plugin

### PR DESCRIPTION
**Pull request by iFlameing**
Fixes #230 
Please! comment if I am missing something.

### CODE REVIEW

File: docs/reference/configuration.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        logLevel: 'DEBUG',
+        websocketUpdates: true || false,
+      },
+    },
+  ]
+```
+* baseUrl : Points to your Plone CMS Url
+* token : If you provide token as options plugin will be able to fetch our private field data also.
+* logLevel : This will provide additional logging for the plugin. Basically this will provide a debugging environment.
+* websocketUpdates: You can set websocketUpdates as true and false. True will enable the Gatsby Preview for your site. It shows you instant update when you made some changes to Plone CMS.
```
Comment (datakurre) [inline docs/reference/configuration.md]: Actually there are some options, but these are the "stable" ones.

Could you use table for the parameters and describe them before the example like in https://github.com/gatsbyjs/gatsby/tree/master/themes/gatsby-theme-blog#usage

There may be many styles in the current documentation, but probably it is good to try to move towards other Gatsby projects.

Comment (iFlameing) [reply docs/reference/configuration.md]: ok! I will add a table, explaining all the config options as you suggested :) thanks!

Comment (iFlameing) [reply docs/reference/configuration.md]: I have made changes, please! comment if I am missing something :)

Comment (iFlameing) [reply docs/reference/configuration.md]: Since pr is merged I am resolving this conversation :)

**Pull request merged**